src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -362,7 +362,7 @@ program. A program needs several types of metadata that you create in the **Main
 | **Completed events expiry days** |  Defines the number of days for which you can edit a completed event. This means that when an event is completed and the specified number of expiry days has passed, the event is locked. <br> <br> If you set "Completed events expiry days" to 10", an event is locked ten days after the completion date. After this date you can no longer edit the event.  |
 | **Expiry period type**<br> <br>**Expiry days** |  The expiry days defines for how many days after the end of the previous period, an event can be edited. The period type is defined by the expiry period type. This means that when the specified number of expiry days has passed since the end date of the previous period, the events from that period are locked. <br> <br> If you set the expiry type to "Monthly" and the expiry days to "10" and the month is October, then you can't add or edit an event to October after the 10th of November.  |
 | **Minimum number of attributes required to search** |  Specify the number of tracked entity attributes that needs to be filled in to search for Tracked Entities in the Program.  |
-| **Maximum number of tracked entity instances to return in search** |  Specify the maximum number of tracked entity instances that should be returned in a search. Enter 0 for no limit.  |
+| **Maximum number of tracked entities to return in search** |  Specify the maximum number of tracked entities that should be returned in a search. Enter 0 for no limit. More info [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).|
```

Comment:
```suggestion
| **Maximum number of tracked entities to return in search (outside capture scope)** |  Specify the maximum number of tracked entities that is allowed to match a search outside capture scope. This setting can be used to force the user to search in a more targeted manner and narrow down the search results, rather than allowing a user to search and see a large number of results outside their capture scope. Enter 0 for no limit. More info [here](../developer/web-api/tracker.md#tracked-entities-collection-limits).|
```